src/libstd/time/duration.rs

```diff
@@ -45,6 +45,7 @@ macro_rules! try_opt {
 
 /// ISO 8601 time duration with nanosecond precision.
 /// This also allows for the negative duration; see individual methods for details.
+#[unstable(feature = "std_misc")]
```

Comment:
Adding `#![feature(std_misc)]` seems to already make this type available, did some code you have not compile with the feature gate?